atlas-db-controller/controller.go

```diff
@@ -295,22 +297,28 @@ func (c *Controller) syncSchema(key string) error {
 	dbName := schema.Spec.Database
 	db, err := c.dbsLister.Databases(namespace).Get(dbName)
 	if err != nil {
-		runtime.HandleError(fmt.Errorf("cannot get database `%s`: %s", dbName, err))
+		schemaStatusMsg = fmt.Sprintf("cannot get database `%s`: %s", dbName, err)
+		c.updateDatabaseSchemaStatus(key, schema, StateError, schemaStatusMsg)
+		runtime.HandleError(fmt.Errorf(schemaStatusMsg))
 		return err
 	}
 
 	glog.Info(db.Spec.ServerType)
 	glog.Info(db.Spec.Dsn)
 
 	if db.Spec.ServerType != "postgres" { //  && db.Spec.ServerType != ...
-		err = fmt.Errorf("unsupported database server type `%s`", db.Spec.ServerType)
+		schemaStatusMsg = fmt.Sprintf("unsupported database server type `%s`", db.Spec.ServerType)
+		c.updateDatabaseSchemaStatus(key, schema, StateError, schemaStatusMsg)
```

Comment:
Can you add supported list as well.

Reply:
right now it just doing a string check with postgres. Need to reimplement this with list of supported type dbs